server/shared/configFetcher.go

```diff
@@ -26,6 +26,7 @@ func NewConfigFetcher(remote remote.Remote, user *model.User, repo *model.Repo,
 
 func (cf *configFetcher) Fetch() (files []*remote.FileMeta, err error) {
 	var file []byte
+	cf.repo.Config = strings.TrimSpace(cf.repo.Config)
```

Comment:
This should be done at in post func saving the string to the database.

Reply:
oh sorry did add int unintentional should be in a extra pull :) - will rm it